ocw/lib/k8s.py

```diff
@@ -1,9 +1,10 @@
 from datetime import datetime, timezone
-from kubernetes.client import BatchV1Api
+from kubernetes.client import BatchV1Api, CoreV1Api
 from ocw.lib.provider import Provider
 
 
-def clean_jobs(provider: Provider, client: BatchV1Api, cluster_name: str):
+def clean_jobs(provider: Provider, client, cluster_name: str):
+    client = client.BatchV1Api()
```

Comment:
The `client` is defined in `ocw/lib/{aks,eks,gke}.py` as:
```diff
- self.__kubectl_client[cluster_name] = kubernetes.client.BatchV1Api()
+ self.__kubectl_client[cluster_name] = kubernetes.client
```
because of I need both `BatchV1Api()` in `clean_jobs` as well as `CoreV1Api()` in `clean_helm_namespaces`.
Would you have an idea how to improve that?

Reply:
I've tried to do some improvements.